contracts/Stake/StakeManager.sol

```diff
@@ -314,7 +325,8 @@ contract StakeManager is Initializable, Ownable {
             "StakeManager:undelegateStash - Only staker can undelegate stash"
         );
         require(
-            _stash.delegatedCluster != address(0),
+            _stash.delegatedCluster != address(0) && 
+            _stash.delegatedCluster != address(0xFFfFfFffFFfffFFfFFfFFFFFffFFFffffFfFFFfF),
```

Comment:
In case of undelegated if address is 0x00, we  can just skip  undelegation call I guess.